CMakeLists.txt

```diff
@@ -343,7 +343,9 @@ if (INTERN_BUILD_MOBILE AND NOT BUILD_CAFFE2_MOBILE)
   set(FEATURE_TORCH_MOBILE ON)
   set(NO_API ON)
   set(USE_FBGEMM OFF)
-  set(USE_PYTORCH_QNNPACK ON)
+  if (NOT DEFINED USE_PYTORCH_QNNPACK)
```

Comment:
I think we set default value for "USE_PYTORCH_QNNPACK" on Line 181 - will NOT DEFINED ever become true?
I think we should simply removed "set(USE_PYTORCH_QNNPACK ON)" here as we already have default value set to ON for it. I cannot remember why I set this in the first place, maybe was trying to enforce using it everywhere for mobile.

Reply:
That's the best way to do it.